### ISSUE TITLE
dew_dun02 doesn't exist

### ISSUE THREAD

**Issue by Vespertine32** [OpenKore/openkore#1752]
i set my bot to go to dew_dun02 from dewata town, however my bot gives this error "invalid map specified for lockmap - map dew_dun02 doesn't exist".

I have my portal set up
```
dewata 44 252 dew_fild01 373 212
dew_dun02 305 30 dew_fild01 48 65
dew_fild01 48 65 dew_dun02 302 30
```

I also added this on my maps-custom
`dewata.rsw#Unknown Area#`

I also have these files in the field folder.
```
dew_dun01.fld
dew_dun01.dist
dew_dun02.fld
dew_dun02.dist
dewata.fld
dewata.dist
```

Still not working despite having all these.
Please help.

**Comment by c4c1n6kr3m1** [comment]
you put dewata so bot recognize dewata
but not dew_dun02

**Comment by Vespertine32** [comment]
bot is saved at dewata town, so a portal to alberta to dewata still necessary?
i also added these to maps-custom, but the bot still not recognizes the dungeon from dewata.
```
dew_dun02#Dewata Dungeon#
dew_dun01#Dewata Dungeon#
```

**Comment by c4c1n6kr3m1** [comment]
rsw

**Comment by alisonrag** [comment]
```
dew_dun01.rsw#Dewata Dungeon 01#
dew_dun02.rsw#Dewata Dungeon 02#
dewata.rsw#Dewata City#
```

**Comment by Vespertine32** [comment]
sorted! thanks so much

**Comment by sctnightcore** [comment]
Fix : #1831 1831
